crates/containerd-shim-wasm/Cargo.toml

```diff
@@ -42,13 +42,5 @@ ttrpc-codegen = { version = "0.4.2", optional = true }
 tempfile = "3.8"
 
 [features]
-default = []
-libcontainer_default = ["libcontainer/default"]
```

Comment:
This would mean that shim-implementors would be required to use `libcontainer`? Should the `Instance` api still be available for workloads that don't want to use libcontainer? Is that a scenario we should support?

Reply:
It would be a required dependency in Linux, only.
You can still choose not to use it by using the `Instance` API instead of the `Engine` API.
The main downside of this is that we are forcing downstream dependencies to build libcontianer even if they don't use it. The linker should be smart enough to remove almost every trace of libcontainer from the final binary (except for the error definitions).

The main motivation for this is that the features system is not great with system dependent branching, and the number of combinations we have to handle grows exponentially.
But I'm open to suggestions